cmd/ocm-backplane/console/console_test.go

```diff
@@ -450,19 +463,61 @@ var _ = Describe("console command", func() {
 			ocmEnvironment, _ := cmv1.NewEnvironment().BackplaneURL("fakeBackPlaneUrl").Build()
 			// Tell mockOCM interface to return ocnEnvironment
 			mockOcmInterface.EXPECT().GetOCMEnvironment().Return(ocmEnvironment, nil).AnyTimes()
-			o := consoleOptions{terminationFunction: &execActionOnTermMockStruct{}}
+			o := newConsoleOptions()
+			o.terminationFunction = &execActionOnTermMockStruct{}
 			o.port = "1337"
 			o.url = "http://127.0.0.2:1447"
 			o.openBrowser = false
 			o.containerEngineFlag = DOCKER
 			ce, err := o.getContainerEngineImpl()
 			Expect(err).To(BeNil())
-			done := make(chan bool)
+
 			errs := make(chan error)
-			o.runContainers(ce, done, errs)
+			o.runContainers(ce, errs)
+
 			Expect(errs).To(BeEmpty())
 			os.Setenv("BACKPLANE_DEFAULT_OPEN_BROWSER", "")
-			removePath(oldpath)
+			setPath(oldpath)
 		})
 	})
 })
+
+type execActionOnTermMockStruct struct{}
+
+func (e *execActionOnTermMockStruct) execActionOnTerminationFunction(action postTerminateFunc) error {
+	return action()
+}
+
+func createPathPodman() string {
+	return createPath("podman")
+}
+
+func createPathDocker() string {
+	return createPath("docker")
+}
+
+// TODO: this creates a local tmp bin dir and puts files in it so tests find a given binary (docker/podman etc).
+// Refactor those test sites to utilize a fake filesystem or other mechanism that doesn't rely on local state.
+func createPath(binary string) string {
```

Comment:
Copied this from the `console_tests_util.go` file and factored it out to reduce code duplication.